src/js/actions/menu.js

```diff
@@ -75,14 +75,18 @@ define(function (require, exports) {
         }
 
         var isPlaceCommand = payload.commandID === _PLACE_LINKED_MENU_ID ||
-                payload.commandID === _PLACE_EMBEDDED_MENU_ID;
+            payload.commandID === _PLACE_EMBEDDED_MENU_ID;
```

Comment:
any reason of cancelling the indent? this looks like we are defining two variables